backend/migrations/versions/799a2343955e_fix_moduletype_enum.py

```
@@ -0,0 +1,118 @@
+        "RECOGNITION_GUIDE": "recognition_guide",
+        "QUIZ": "quiz",
+        "CHALLENGE": "challenge",
+        "CHALLENGE_SOLUTION": "solution_guide",
+        "BONUS_CHALLENGE": "bonus_challenge",
+        "BONUS_SOLUTION": "bonus_solution_guide",
+    }
+
+    # Update the temporary column with the old enum values
+    for new_value, old_value in mapping.items():
+        op.execute(
+            f"UPDATE modules SET module_type_temp = '{old_value}' WHERE module_type = '{new_value}'"
```

Ruff (0.8.2)

Possible SQL injection vector through string-based query construction

(S608)